src/views/catalog/CreateVMHorizontalNav/components/CreateFromInstanceTypes/components/AddBootableVolumeButton/components/AddBootableVolumeModal/components/FilterSelect/utils/utils.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { ReactElement } from 'react';
+
+import { SelectOption } from '@patternfly/react-core';
+
+export const getSelectOption = (optionLabel: string) => (name: string) =>
```

Comment:
hmm... get*** is misleading in this case, as this function eventually returns a component.
util files usually don't return elements/components.